client/my-sites/site-settings/transfer-site-to-user/domain-transfer-instructions.jsx

```diff
@@ -0,0 +1,64 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'my-sites/site-settings/action-panel';
+import ActionPanelTitle from 'my-sites/site-settings/action-panel/title';
+import ActionPanelBody from 'my-sites/site-settings/action-panel/body';
+import ActionPanelFooter from 'my-sites/site-settings/action-panel/footer';
+import Button from 'components/button';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteDomain } from 'state/sites/selectors';
+
+class DomainTransferInstructions extends PureComponent {
+	static propTypes = {
+		selectedSiteDomain: PropTypes.string,
+		onCancel: PropTypes.func,
+		translate: PropTypes.func,
+	};
+
+	render() {
+		const translate = this.props.translate;
+
+		return (
+			<ActionPanel>
+				<ActionPanelBody>
+					<ActionPanelTitle>{ translate( 'Your Site Has A Domain' ) }</ActionPanelTitle>
+					<p>{ translate( 'You must transfer your domain first!' ) }</p>
+				</ActionPanelBody>
+				<ActionPanelFooter>
+					<Button
+						onClick={ this.props.onCancel }
+						className="transfer-site-to-user__cancel is-scary"
+					>
+						<Gridicon icon="cross" size={ 48 } />
+						{ translate( 'Cancel' ) }
+					</Button>
+					<Button className="transfer-site-to-user__continue">
+						{ translate( 'Transfer Your Domain' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Transfer Domain' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-transfer)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).